subsys/bluetooth/services/fast_pair/Kconfig.fast_pair

```diff
@@ -29,6 +29,12 @@ config BT_FAST_PAIR_PN
 	help
 	  Enable Fast Pair Personalized Name extension.
 
+config BT_FAST_PAIR_CHRC_MODEL_ID
+	bool "Fast Pair Model ID characteristic"
+	default y if !BT_FAST_PAIR_FMDN
+	help
+	  Use the Fast Pair Model ID characteristic.
+
```

Comment:
Maybe it could be named `BT_FAST_PAIR_GATT_SERVICE_MODEL_ID` and put under `if BT_FAST_PAIR_GATT_SERVICE`. It should be aligned with the naming and placement of `BT_FAST_PAIR_GATT_SERVICE_ADDITIONAL_DATA`.